lib/manageiq/release/repo.rb

```diff
@@ -38,9 +35,9 @@ def fetch
         git.fetch(:all => true, :tags => true)
       end
 
-      def checkout(branch)
+      def checkout(branch, source = "origin/#{branch}")
```

Comment:
This is the second time that "origin" is actually a meaningful name :laughing: :trollface: